packages/os/disable-udp-offload.service

```diff
@@ -0,0 +1,22 @@
+# This unit disables UDP offload for the default primary interface (eth0) in
+# VMware.  This avoids an issue where newer kernels (5.10+) and the vmxnet3
+# driver conflict with NSX-T.
+[Unit]
+Description=Disables UDP offload
+After=network-online.target
+# Run before bootstrap containers so a user could reverse the change via
+# bootstrap container if they desired.
+Before=bootstrap-containers@.service
```

Comment:
Is this syntax correct? You're using it to mean "before all units instantiated from this template" but I'm surprised that would be valid.

In any case, running before bootstrap containers is implied by `preconfigured.target`, so this isn't necessary and should be omitted.

Reply:
Ah - you're absolutely right - thanks!